worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -873,6 +851,92 @@ public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
         return null;
     }
 
+    private boolean noBlacklistTrimXZ(
+            int lowerX,
+            int upperX,
+            int lowerZ,
+            int upperZ,
+            int upperZi,
+            int lowerZi,
+            boolean trimX,
+            boolean trimZ,
+            char[] charArr,
+            int[] intArr
+    ) {
+        if (!(trimX || trimZ)) {
+            return true;
+        }
+        int indexY = 0;
+        if (charArr == null) {
```

Comment:
Add comments to signify this explicitly means that intArr is not null (might be better to check `intArr != null` for readability) ... or use two different methods as mentioned above